client-reactjs/src/components/layout/Header.js

```diff
@@ -128,6 +128,7 @@ class AppHeader extends Component {
     ) {
       this.props.dispatch(applicationActions.getClusters());
       this.props.dispatch(applicationActions.getConsumers());
+      this.props.dispatch(applicationActions.getLicenses());
```

Comment:
We fetch static licenses list each time we open file view. we can get them once on initial load, and store it in redux, under aplication. will save us a trip in file and file-template request.